tests/templates/test_state_preparations/test_basis_state_prep.py

```diff
@@ -76,6 +76,34 @@ def circuit():
 
         assert np.allclose(output_state, target_state, atol=tol, rtol=0)
 
+    @pytest.mark.jax
+    @pytest.mark.parametrize(
+        "basis_state,wires,target_state",
+        [
+            ([0, 1], [0, 1], [0, 1, 0]),
+            ([1, 1, 0], [0, 1, 2], [1, 1, 0]),
```

Comment:
Let's add some non-std wires too, like randomly ordered wires, wires of custom wire labels, etc.

Reply:
I added a case for randomly-ordered wires. The use of custom wire labels isn't "allowed" by these tests because we use the `qubit_device_3_wires` fixture.